tests/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ if(LIBEVSE_CRYPTO_SUPPLIER_OPENSSL)
     add_compile_definitions(LIBEVSE_CRYPTO_SUPPLIER_OPENSSL)
 endif()
 
+add_compile_definitions(BUILD_TESTING_EVSE_SECURITY)
```

Comment:
Shouln't this be optional?

Reply:
It is in 'tests/CmakeList.txt' and in the Cmake in a folder up, the dir 'tests' is conditionally added:
https://github.com/EVerest/libevse-security/blob/153a1d902a5e0902772dcd38749bf8f9bb6bac9e/CMakeLists.txt#L60

Reply:
Ah I missed that thanks